optimum/amd/ryzenai/configs/vaip_config.json

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
Can we clarify with AMD the license compatibility of this file? I think the license is https://account.amd.com/content/dam/account/en/licenses/download/eula-beta-software.pdf which may not be compatible.

Reply:
I think this file here is copied from `voe-4.0-wind_amd64`. Alternatively https://github.com/amd/RyzenAI-SW/blob/main/example/Ryzen-AI-Library/windows/vaip_config.json may be used which I guess is under MIT (?) ?

Reply:
I have asked the question to Sumit.

Although the config is same as https://github.com/amd/RyzenAI-SW/blob/main/tutorial/yolov8_e2e/vaip_config.json, which is under MIT license. So should not be an issue.

Reply:
yes we are releasing this out, so it's ok to have it here too.